guardian/cdk-cli
# Add CI github workflow

## What does this change?

This PR adds a GitHub workflow for CI. It runs the following against node version `14.15.1`:

* `scripts/build.sh`
* `scripts/lint.sh`
* `scripts/test.sh`

## How to test

We should see this workflow run immediately in this PR. Check the Actions tab to see it run (hopefully successfully).
